tests/PullRequest.py

```diff
@@ -160,10 +161,30 @@ def testEditWithAllArguments(self):
+        self.assertListKeyEqual(self.pull.get_files(), lambda f: f.filename,
+                                ["codegen/templates/GithubObject.py", "src/github/AuthenticatedUser.py",
+                                 "src/github/Authorization.py", "src/github/Branch.py", "src/github/Commit.py",
+                                 "src/github/CommitComment.py", "src/github/CommitFile.py", "src/github/CommitStats.py",
+                                 "src/github/Download.py", "src/github/Event.py", "src/github/Gist.py",
+                                 "src/github/GistComment.py", "src/github/GistHistoryState.py",
+                                 "src/github/GitAuthor.py", "src/github/GitBlob.py", "src/github/GitCommit.py",
+                                 "src/github/GitObject.py", "src/github/GitRef.py", "src/github/GitTag.py",
+                                 "src/github/GitTree.py", "src/github/GitTreeElement.py", "src/github/Hook.py",
+                                 "src/github/Issue.py", "src/github/IssueComment.py", "src/github/IssueEvent.py",
+                                 "src/github/Label.py", "src/github/Milestone.py", "src/github/NamedUser.py",
+                                 "src/github/Organization.py", "src/github/Permissions.py", "src/github/Plan.py",
+                                 "src/github/PullRequest.py", "src/github/PullRequestComment.py",
+                                 "src/github/PullRequestFile.py", "src/github/Repository.py",
+                                 "src/github/RepositoryKey.py", "src/github/Tag.py", "src/github/Team.py",
+                                 "src/github/UserKey.py", "test/Issue.py", "test/IssueEvent.py",
+                                 "test/ReplayData/Issue.testAddAndRemoveLabels.txt",
+                                 "test/ReplayData/Issue.testDeleteAndSetLabels.txt",
+                                 "test/ReplayData/Issue.testGetLabels.txt", "test/ReplayData/IssueEvent.setUp.txt"])
 
```

Comment:
All of these changes are beside the point of your change. Personally, I'd prefer in future if you split out things like this into a clean up commit.

Reply:
Agreed. This was added accidentally during formatting. Reverting back to original.